### PR TITLE
Fix MMF Momentum Feedback

### PR DESCRIPTION
**Pull request by whannah1**
Fix the feedback of momentum for 3D MMF simulations. The momentum feedback can be enabled with the MMF_MOMENTUM_FEEDBACK preprocessor variable. Note that this also disables the CRM  "dosurface" option that applies friction in the CRM. Also add MMF_TKEW history variable.

[BFB]

### CODE REVIEW

File: components/eam/src/physics/crm/sam/crm_module.F90 [hunk 1]
```diff
@@ -1231,6 +1218,10 @@ subroutine crm(lchnk, ncrms, dt_gl, plev,       &
       crm_output%qltend (icrm,k) =      (qln  (icrm,k) - crm_input%ql  (icrm,k)) * icrm_run_time
       crm_output%qcltend(icrm,k) =      (qccln(icrm,k) - crm_input%qccl(icrm,k)) * icrm_run_time
       crm_output%qiltend(icrm,k) =      (qiiln(icrm,k) - crm_input%qiil(icrm,k)) * icrm_run_time
+#if defined(MMF_MOMENTUM_FEEDBACK)
```
Comment (lee1046) [inline components/eam/src/physics/crm/sam/crm_module.F90:1221]: Is the vertical index k missing in crm_output%ultend and crm_output%vltend ?

Comment (whannah1) [reply components/eam/src/physics/crm/sam/crm_module.F90:1221]: good catch, I'll fix it.

**Review by lee1046 — APPROVED** [review]
Everything looks good to me. Good to merge as it is.

**Comment by brhillman** [comment]
Merged to next.

**Comment by brhillman** [comment]
Merged to master.

**Pull request merged**